app/panel/components/Summary.jsx

```diff
@@ -219,15 +220,24 @@ class Summary extends React.Component {
 		this.props.history.push(this._isPlusSubscriber() ? '/subscription/info' : `/subscribe/${!!this.props.user}`);
 	}
 
+	/**
+	 * Show the Rewards view
+	 * Used to handle user clicking on the Rewards icon
+	 */
+	showRewardsView() {
```

Comment:
Do you use this or do you rely on the `NavButton` link?  I'm noticing a bug when I click the Rewards button but not the SVG.  It takes me to a blank Detail View.

Reply:
Fixed showRewardsView (renamed to showRewardsList) to correctly hanle click and convert the NavButtons to ReactSVGs to simplify & give showRewardsList sole responsibility.

Reply:
Please revert to use the `NavButton`.  That way worked if you had only removed the `onClick` handler.  This new implementation introduces two bugs:
1. There is no `cursor: pointer` on the SVG.  There should be because it makes sense as you can click the SVG and also because the Historical Stats link above has it.
2. You can click the box around the SVG while you can't for Historical Stats.

Please make these as similar as possible by reverting to the old way of using `NavButton` (like the Historical Stats button does) and by removing the `onClick` handler (which again, the Historical Stats button doesn't have).

Reply:
I've made the UIs and implementations of the Historical Stats and Rewards navicons consistent, but by converting the Stats navicon to use ReactSVG also instead of reverting to NavButton for the Rewards navicon. I chose this solution because NavButton isn't sufficient to handle clicks on the Rewards navicon, which need to call toggleExpert in addition to pushing a new route onto the history stack. I think letting user click anywhere inside the bordered navicon area is, anyhow, more user-friendly than requiringt them to click on the icon itself. It's also more consistent with the click-on-donut behavior (clicking on empty space inside the donut works as well as clicking the donut itself). The cursor behavior is now consistent between the two navicons also.